Sources/BetterCodable/LossyDictionary.swift

```diff
@@ -81,6 +82,21 @@ extension LossyDictionary: Decodable where Key: Decodable, Value: Decodable {
 
         self.wrappedValue = elements
     }
+
+    private static func extractKeys(
+        from decoder: Decoder,
+        container: KeyedDecodingContainer<DictionaryCodingKey>
+    ) throws -> [(DictionaryCodingKey, String)] {
+        // Decode a dictionary ignoring the values to decode the original keys
+        // without using the `JSONDecoder.KeyDecodingStrategy`.
+        let keys = try decoder.singleValueContainer().decode([String: AnyDecodableValue].self).keys
+
+        return zip(
+            container.allKeys.sorted(by: { $0.stringValue < $1.stringValue }),
+            keys.sorted()
```

Comment:
I understand why this is necessary, but how robust is it? Can you think of a scenario where the sorting would be different when converting from snake case? I'd hate for there to be an edge case that's missed.

I think the answer is "no", but is it possible to route through this only when the key strategy isn't the default?

Reply:
Thanks for the due diligence. I'll play around with this tonight and try and button up some other in flight work.